sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -923,7 +923,8 @@ glsTextureTestUtil.computeLodFromDerivates3D = function(mode, dudx, dvdx, dwdx,
             DE_ASSERT(false);
     }
 
-    return Math.log(p) * 1.44269504089;
+    // Native dEQP uses 32-bit numbers. So here 64-bit floating numbers should be transformed into 32-bit ones to ensure the correctness of the result.
+    return new Float32Array([Math.log(p)])[0] * new Float32Array([1.44269504089])[0];
```

Comment:
Yes. It is really a big difference between C++ and JS and we should pay much attention on it when trying to translate C++ codes to JS ones.